RevenueCatUI/Templates/Template2View.swift

```diff
@@ -83,20 +86,35 @@ struct Template2View: TemplateViewType {
                 Spacer()
             }
 
-            if self.configuration.mode.shouldDisplayPackages {
-                self.packages
+            VStack {
+                self.packagesScrollHack
+
                 Spacer()
-            } else {
-                self.packages
-                    .onSizeChange(.vertical) { if $0 > 0 { self.containerHeight = $0 } }
-                    .hideFooterContent(self.configuration,
-                                       hide: !self.displayingAllPlans,
-                                       offset: self.containerHeight)
             }
+            // Sets a max footer height so there is enough room for paywall content above
+            .frame(maxHeight: configuration.mode == .fullScreen ? nil : Constants.maxFooterHeight)
```

Comment:
Why not use `.infinite`? I think it accomplishes the same goal without hardcoding a magic number.

Reply:
Removed this. We can add a parameter for the maximum height if users request it.